README.md

```diff
@@ -6,11 +6,11 @@
 
 A React router library enabling the render-as-you-fetch concurrent UI pattern.
 
-* [Overview](#overview)
-* [Getting Started](#getting-started)
-* [Usage Examples](#usage-examples)
-* [API reference](#api-reference)
-* [License](#license)
+- [Overview](#overview)
```

Comment:
Curious why this got automatically changed and also wonder whether we choose one over other here as a convention.

Reply:
Possibly automated with someone's formatting tools...